src/env/test/mod.rs

```diff
@@ -70,6 +86,12 @@ impl TestEnv {
     pub fn customization_mut(&mut self) -> &mut CustomizationImpl {
         &mut self.customization
     }
+
+    pub fn seed_rng_from_u64(&mut self, state: u64) {
```

Comment:
I would make this a method of `TestRng256` to get better organization, i.e. `env.rng().seed_from_u64(42)` instead of `env.seed_rng_from_u64(42)`.

Reply:
Done